emacs/radian.el

```diff
@@ -5352,40 +5374,40 @@ your local configuration."
 ;; We should only get here if init was successful. If we do,
 ;; byte-compile this file asynchronously in a subprocess using the
 ;; Radian Makefile. That way, the next startup will be fast(er).
-(run-with-idle-timer
- 1 nil
- #'radian-byte-compile)
+(when (radian-enabled-p bytecomp)
```

Comment:
Is `bytecomp` a real package? When people say "package" sometimes they mean package as in "could be installed by `package.el` or `straight.el`" and sometimes they mean package as in "could be loaded using `require`" (I would call the latter a "feature").

What's the intent in this pull request---that you can disable packages, or that you can disable features? I think I'd suggest the former, since some packages provide many features which don't make a lot of sense to disable individually, in which case `(radian-enabled-p bytecomp)` doesn't make a ton of sense.

Reply:
Thanks for the comments, I will go through them this week. 

To answer your question, I guess I am thinking of the latter, i.e., disabling features (I called them packages because of `use-package`). Perhaps I should rename the option to `radian-disabled-features`? I think having this option for packages *and* features makes sense because it makes Radian more modular and provides more fine control of its config. For example, I personally disable the `bytecomp` feature that Radian loads because it is unnecessary in my setup.

Your point of a single package providing many features is true, but I am not sure having a dependency tree of packages/features (so that disabling a "package" disables all its "features") would be worth the effort considering that I have not personally run into problems with the simple implementation in this PR.

Reply:
Fair enough. I think it's fine to do it on a feature basis, and adjust later in case problems turn up for some reason. Your reasoning makes sense.